processing/data_collection/gazette/spiders/mg_governador_valadares.py

```diff
@@ -0,0 +1,94 @@
+import ast
+from datetime import datetime, date
+import json
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgGovernadorValadares(BaseGazetteSpider):
+    TERRITORY_ID = "3127701"
+    name = "mg_governador_valadares"
+    allowed_domains = ["valadares.mg.gov.br"]
+    start_urls = [
+        "https://www.valadares.mg.gov.br/ajaxpro/diel_diel_lis,App_Web_eozglfst.ashx"
+    ]
+
+    current_page = 0
+    ITEMS_PER_PAGE = 10
+
+    def start_requests(self):
+        url = self.start_urls[0]
+        yield self.make_request(url, self.current_page)
+
+    def make_request(self, url, page):
+        return scrapy.Request(
+            url,
+            method="POST",
+            headers={"X-AjaxPro-Method": "GetDiario",},
+            body=self.make_body(page),
+            dont_filter=True,
```

Comment:
Is there a specific reason of why the request cannot be filtered?

Reply:
I think the author used it because it is a lot of requests to the same URL, but I don't know if it is needed. I will look into it